### ISSUE TITLE
Compilation failure in Linux: Invalid CEN header

### ISSUE THREAD

**Issue by aberenguel** [sepinf-inc/IPED#2060]
Using Ubuntu 22.04
```sh
mvn -version
Apache Maven 3.6.3
Maven home: /usr/share/maven
Java version: 11.0.22, vendor: BellSoft, runtime: /home/berenguel/opt/java/liberica/jdk-11.0.22-full
Default locale: pt_BR, platform encoding: UTF-8
OS name: "linux", version: "6.5.0-14-generic", arch: "amd64", family: "unix"
```

After the merge in commit 81aec56e9, it is occurring this failure when compiling the project with Maven.
```
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.10.1:compile (default-compile) on project iped-parsers-impl: Compilation failure: Compilation failure: 
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/shareaza/XMLAttribute.java:[19,1] cannot access iped.parsers.shareaza
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/WAContact.java:[1,1] cannot access iped.parsers.whatsapp
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/evtx/model/EvtxName.java:[1,1] cannot access iped.parsers.evtx.model
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/standard/StandardParser.java:[19,1] cannot access iped.parsers.standard
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/winx/TimelineEntry.java:[1,1] cannot access iped.parsers.winx
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/telegram/MapTypeMSG.java:[19,1] cannot access iped.parsers.telegram
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/fork/MemoryURLStreamHandlerFactory.java:[17,1] cannot access iped.parsers.fork
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/edge/EdgeWebCacheException.java:[1,1] cannot access iped.parsers.browsers.edge
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/util/ISO6709Extractor.java:[17,1] cannot access iped.parsers.util
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/skype/SkypeMessage.java:[1,1] cannot access iped.parsers.skype
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariSqliteParser.java:[1,1] cannot access iped.parsers.browsers.safari
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/jdbc/TableReportGenerator.java:[1,1] cannot access iped.parsers.jdbc
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/sqlite/SQLiteUndeleteTableResultSetAdapter.java:[1,1] cannot access iped.parsers.sqlite
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/security/CryptoAPIBlobParser.java:[1,1] cannot access iped.parsers.security
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/discord/json/DiscordEmbed.java:[1,1] cannot access iped.parsers.discord.json
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/usnjrnl/UsnJrnlParser.java:[1,1] cannot access iped.parsers.usnjrnl
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/gdrive/GDriveCloudGraphParser.java:[1,1] cannot access iped.parsers.gdrive
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/bittorrent/TorrentFileParser.java:[1,1] cannot access iped.parsers.bittorrent
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/eventtranscript/CensusEntry.java:[1,1] cannot access iped.parsers.eventtranscript
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/emule/KnownMetDecoder.java:[1,1] cannot access iped.parsers.emule
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/skype/v8/SkypeMessageV12.java:[1,1] cannot access iped.parsers.skype.v8
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/mail/win10/tables/StoreTable.java:[1,1] cannot access iped.parsers.mail.win10.tables
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/misc/OFCParser.java:[2,1] cannot access iped.parsers.misc
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/mail/MSGParser.java:[1,1] cannot access iped.parsers.mail
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/registry/RegRipperParser.java:[1,1] cannot access iped.parsers.registry
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/registry/model/HiveCell.java:[1,1] cannot access iped.parsers.registry.model
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/evtx/template/TemplateValueDescriptor.java:[1,1] cannot access iped.parsers.evtx.template
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/AbstractSqliteBrowserParser.java:[1,1] cannot access iped.parsers.browsers
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/security/capi/CapiBlob.java:[1,1] cannot access iped.parsers.security.capi
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/mail/win10/entries/MessageEntry.java:[1,1] cannot access iped.parsers.mail.win10.entries
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/lnk/LNKShellItemFileEntry.java:[1,1] cannot access iped.parsers.lnk
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/firefox/FirefoxSqliteParser.java:[1,1] cannot access iped.parsers.browsers.firefox
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/compress/RARParser.java:[19,1] cannot access iped.parsers.compress
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ares/AresEntry.java:[1,1] cannot access iped.parsers.ares
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/threema/ThreemaContact.java:[1,1] cannot access iped.parsers.threema
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/evtx/EvtxParser.java:[1,1] cannot access iped.parsers.evtx
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/video/FLVParserWrapper.java:[1,1] cannot access iped.parsers.video
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/discord/DiscordHTMLReport.java:[1,1] cannot access iped.parsers.discord
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/apk/CustomApkFile.java:[1,1] cannot access iped.parsers.apk
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/vcard/VCardParser.java:[1,1] cannot access iped.parsers.vcard
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ocr/OCRParser.java:[19,1] cannot access iped.parsers.ocr
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/discord/cache/CacheAddr.java:[1,1] cannot access iped.parsers.discord.cache
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/registry/keys/AresKeyParser.java:[1,1] cannot access iped.parsers.registry.keys
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/com/whatsapp/MediaData.java:[1,1] cannot access com.whatsapp
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/chrome/ChromeSqliteParser.java:[1,1] cannot access iped.parsers.browsers.chrome
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/python/PythonParser.java:[1,1] cannot access iped.parsers.python
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/ufed/ReportGenerator.java:[1,1] cannot access iped.parsers.ufed
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/external/ExternalParser.java:[17,1] cannot access iped.parsers.external
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/database/MSAccessParser.java:[19,1] cannot access iped.parsers.database
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/mail/win10/ColumnCodes.java:[1,1] cannot access iped.parsers.mail.win10
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/image/TiffPageParser.java:[1,1] cannot access iped.parsers.image
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/sqlite/detector/SQLiteContainerDetector.java:[1,1] cannot access iped.parsers.sqlite.detector
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/evtx/windows/SID.java:[1,1] cannot access iped.parsers.evtx.windows
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/plist/detector/PListDetector.java:[1,1] cannot access iped.parsers.plist.detector
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/ie/IndexDatParser.java:[19,1] cannot access iped.parsers.browsers.ie
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/tor/TorTcParser.java:[1,1] cannot access iped.parsers.tor
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/fork/ContentHandlerResource2.java:[28,1] cannot access iped.parsers
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/fork/RecursiveMetadataContentHandlerResource.java:[31,1] cannot access iped
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/fork/ParsingTimeout.java:[3,8] cannot access java
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/shareaza/MFCParser.java:[24,8] cannot access org
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/shareaza/MFCParser.java:[24,11] cannot access org.apache
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[12,18] cannot access org.apache.tika
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[12,23] cannot access org.apache.tika.config
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[13,23] cannot access org.apache.tika.exception
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[14,23] cannot access org.apache.tika.extractor
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[16,23] cannot access org.apache.tika.io
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[17,23] cannot access org.apache.tika.metadata
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[19,23] cannot access org.apache.tika.mime
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[20,23] cannot access org.apache.tika.parser
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[22,23] cannot access org.apache.tika.sax
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[24,11] cannot access org.slf4j
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[26,11] cannot access org.xml
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[29,8] cannot access com
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[29,11] cannot access com.dd
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[30,14] cannot access com.dd.plist
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[40,12] cannot access iped.properties
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariPlistParser.java:[42,12] cannot access iped.utils
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariResumedVisit.java:[7,11] cannot access com.google
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/browsers/safari/SafariResumedVisit.java:[7,18] cannot access com.google.common
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/jdbc/JDBCTableReader.java:[33,8] cannot access javax
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/jdbc/JDBCTableReader.java:[35,18] cannot access org.apache.commons
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/jdbc/JDBCTableReader.java:[35,26] cannot access org.apache.commons.codec
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/jdbc/JDBCTableReader.java:[37,23] cannot access org.apache.tika.detect
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/evtx/model/EvtxFile.java:[11,18] cannot access org.apache.lucene
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/evtx/model/EvtxFile.java:[11,25] cannot access org.apache.lucene.util
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/sqlite/SQLite3DBParser.java:[38,11] cannot access org.sqlite
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/sqlite/SQLite3DBParser.java:[40,12] cannot access iped.data
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/sqlite/SQLite3DBParser.java:[45,12] cannot access iped.search
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[23,32] cannot access org.apache.commons.codec.binary
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[24,11] cannot access org.bouncycastle
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[24,24] cannot access org.bouncycastle.crypto
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[25,31] cannot access org.bouncycastle.crypto.macs
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[26,31] cannot access org.bouncycastle.crypto.params
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[28,11] cannot access com.drew
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkExtractor.java:[28,16] cannot access com.drew.lang
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/whatsapp/LinkDownloader.java:[16,26] cannot access org.apache.commons.io
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/discord/json/DiscordReferencedMessage.java:[5,11] cannot access com.fasterxml
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/discord/json/DiscordReferencedMessage.java:[5,21] cannot access com.fasterxml.jackson
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/discord/json/DiscordAuthor.java:[3,29] cannot access com.fasterxml.jackson.annotation
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] /home/berenguel/git/IPED/iped-parsers/iped-parsers-impl/src/main/java/iped/parsers/telegram/Contact.java:[23,8] cannot access dpf
[ERROR]   Invalid CEN header (invalid extra data field size for tag: 0xbfef size: 61373)
[ERROR] -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/MojoFailureException
[ERROR] 
[ERROR] After correcting the problems, you can resume the build with the command
[ERROR]   mvn <args> -rf :iped-parsers-impl
```

**Comment by aberenguel** [comment]
I tried to set `JAVA_TOOL_OPTIONS=-Djdk.util.zip.disableZip64ExtraFieldValidation=true`, but not worked.

**Comment by aberenguel** [comment]
In the parent commit 414b128d1 the problem doesn't happen.

**Comment by lfcnassif** [comment]
I'm also getting this build error when compiling master using Eclipse maven plugin on Windows. Using maven on Windows command line works... Seems the cause was the merge of #1908.

**Comment by lfcnassif** [comment]
Upgrading `com.webcohesion.ofx4j` dependency added to iped-parsers-impl module to last 1.36 version fixed here on Windows. Could you test it on your Linux OS?

**Comment by aberenguel** [comment]
I found out that the problem came from commit ee0a3346c

**Comment by aberenguel** [comment]
> Upgrading `com.webcohesion.ofx4j` dependency added to iped-parsers-impl module to last 1.36 version fixed here on Windows. Could you test it on your Linux OS?

Great! Upgrading to `1.36` solved the failure. Thanks @lfcnassif !

**Comment by lfcnassif** [comment]
Thanks @aberenguel for reporting! I thought it was some corrupted dependency in my local maven repo, but as I was being able to compile the project from command line, I didn't pay the needed attention to the issue. As soon as the remote CI also confirms the fix, I'll close this.

**Comment by lfcnassif** [comment]
Fixed by [1729519](https://github.com/sepinf-inc/IPED/commit/1729519e61fdf90c4f767b7db8aead3211e9eaa7).
